game/view/startmenu.lua

```diff
@@ -29,21 +39,41 @@ local function _initFontValues()
   _width, _height = g.getDimensions()
 end
 
+local function _initLogo()
+  for i = 1, _LOGO_BG_PARTS do
+    _LOGO_BG[i] = RES.loadTexture('logo-bg'..i)
+    _LOGO_BG[i]:setFilter("linear","linear")
+    _LOGO_BG_OFFSET[i] = love.math.random(0,2*math.pi)
+    _LOGO_BG_MAGNITUDE[i] = love.math.random()*1 + 1 --Random float [1,2]
```

Comment:
put these ranges in constants. it took me a while to understand what they meant

Reply:
You meanthe magnitude range right? Because offset only matters if its between 0 and 2*pi since its in radians, and I think 0 and pi are constants enough hahah
I can change the random float "equation" to use a min/max magnitude const.

Reply:
I think what he wants most are meaningful names, such as `SHINY_EFFECT_ANGLE_RANGE_MIN` (ok, maybe not so much). Haven't read the code myself, so I might be off here.